packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.css

```diff
@@ -0,0 +1,37 @@
+lightning-icon{
+  --sds-c-icon-color-foreground-default: white;
+}
+.slds-visual-picker__figure{
+  width: 9.875rem;
```

Comment:
Did you take these values somewhere from a component in the SLDS?

Seems weird to have such fractions of rem;

Reply:
Nope, these values came from the design mockups, I used the same width used in the design and  converting it  from px to rem gave such fractions.